stable/spark/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: spark
-version: 1.0.0
+version: 1.0.1
```

Comment:
Looks like you bumped the wrong version in the `spark` chart. You need to bump the version of the `sparkoperator` chart.

Reply:
Sorry about that. I fixed that.